### DISCUSSION TITLE
博山菜 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by yufree** [yufree/yufree.cn#366] [General]
博山菜 - Miao Yu | 于淼 
 https://yufree.cn/cn/2020/10/04/bo-shan-cai/

**Comment by yufree** [comment]
`Originally posted by 张晶 on 2020-10-14 04:26:59
        `
看到淄博，张店，博山菜，熟悉感扑面而来。我是在山理工读的大学，大学期间第一次去菜馆打工，就是在一家博山菜馆。你上面提到的这些菜都没印象了，我印象中的博山菜竟然是炸昆虫，蚕蛹（到现在也吃不下），鲅鱼馅水饺（大概是我吃到过最好吃的水饺）。\
我印象中的鲁菜，就是盐巴、酱油多放一点。\
我是山西平遥人，现在生活在日本，从谢大看过来的，很喜欢你的文字。

**Reply by yufree** [reply]
`Originally posted on 2020-10-14 13:30:35
        `
我高中就理工大对门读的，很多同学中午去理工大吃饭都说比高中好很多，说起来我还没去过，有机会一定去尝尝。蚕蛹豆虫蚂蚱这些炸昆虫确实比较特色，我最开始也不敢吃，后来才习惯，就当高蛋白食品了，不过听说有些人对昆虫蛋白过敏，吃了会浑身痒。鲅鱼馅水饺其实是胶东菜，不过现在菜馆都是啥好吃做啥了，确实很好吃。

哈哈，确实，很多人说鲁菜核心就是啥都加酱油。

平遥我去过，古镇非常有特色，有机会还会去。谢大可以说是这个网站最大的推荐源了，欢迎常来做客。

**Comment by yufree** [comment]
`Originally posted by yihui on 2021-06-15 21:15:14
        `
这个酥锅感觉跟我这几年做的潍坊蒸鸡有几分相似。蒸鸡的主要材料是鸡和白菜、木耳、香菇，配上各种调料，包括香油、海米、八角、桂皮、花椒水、盐等等，也是一大锅炖烂。本来是蒸菜，但几年前我看老丈人蒸了几次之后，觉得操作太麻烦了，于是趁他们有一次不在家时，我用电高压锅炖了一锅，其实效果也差不多，工作量减少了一大半，还大大省火。

北方菜里这种大锅菜似乎都有放在阳台上结冰的传统。蒸鸡也一样，冻成冰碴，吃的时候挖几勺回来，有些人还对吃冰碴念念不忘，其实也就是你说的稀缺留下的印迹；冰碴的美味在于这一大锅肉的稀缺。

**Reply by yufree** [reply]
`Originally posted on 2021-06-16 02:25:08
        `
哈哈，确实本质就是乱炖。不过酥锅肉里面核心是海带卷五花肉、猪蹄牛肉与鱼肉，鸡肉倒是可选，潍坊淄博本就是邻居，估计这些吃法做法大差不差，就地取材怎么方便怎么来，不过高压锅绝对的入味神器。

我说的上冻倒不是冰碴儿，实际淄博最冷也就零下个位数，一年达到这个温度的天数最多一两周，冬天室外大概0度上下，比北京还是暖和一点的，在东北人眼里都算南方。因为酥锅里有盐，室外低温还不足以冻出冰碴。这里上冻是肉皮冻，因为酥锅里有猪蹄五花肉，只要放凉就会把其他食材跟汤汁冻到一起，而放阳台上大概相当于冰箱冷藏。这样早上煮完挂面再盖上一大勺酥锅，肉冻沾到热面上吃正好给挂面补充滋味还降温。

其实如果不是过年这么舍得放肉做酥锅，平时也可以买些猪皮熬化了加调料花生炖煮，冰箱冷藏过夜后就是花生肉皮冻。这个印象中我家街边熟食店有卖，切成片下酒凉拌正好。照说这个菜胶原蛋白丰富也算个美容菜，但那个年代则是处理食材边角料的小吃凉菜。

**Reply by yufree** [reply]
`Originally posted on 2021-06-16 02:37:09
        `
刚反应过来，谢大是山东女婿？那要是去老丈人那过年或省亲，带上冻带鱼跟成桶的花生油绝对不丢份。

**Reply by yufree** [reply]
`Originally posted by yihui on 2021-06-16 03:31:31
        `
是的，所以我才会研究蒸鸡。嗯，肉皮冻的原理我明白，普通低温下蒸鸡的汤也会变成肉冻（炖的时候需要多一些鸡皮和油，其实我是觉得不是很健康）。而在奥马哈冬天这么冷的地方，蒸鸡放车库里是真的会冻成冰碴。老丈人平日吃饭最怕凉食，但这冰碴鸡照吃不误。

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2021-06-16 03:50:00
        `
冰碴鸡是字面意思的冰碴鸡吗？又冷又油？直接吃？

**Reply by yufree** [reply]
`Originally posted by yihui on 2021-06-16 04:47:00
        `
差不多吧，冷是一定的，而油在炖的过程中都混入汤里了，并不会一大堆油，加上还有素菜吸油，所以不算太腻。要不是从小吃的话，我估计也没有人会习惯吃。我后来做过招待南方客人，端出来别人都不怎么动筷子，而且很惊奇居然吃凉的。

我们老家过年时家家都做一大盆鱼冻（通常是辣的），也是吃凉的，不知你们家是不是也这样。

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2021-06-16 05:04:51
        `
我家那边的凉菜是白砍鸡那种，应该没有更&quot;冷&quot;的了；肉菜带冰渣子的我还是第一次听说。我要是面对这么一盆菜，估计也不敢动筷子：）饮食文化真是博大精深啊，这真是超出我的认知了。

鱼冻，猪皮冻我都没有吃过，应该不是我们那边的常见菜。棒棒鸡，白肉，口水鸡这类凉肉菜比较常见。

**Reply by yufree** [reply]
`Originally posted by yihui on 2021-06-16 05:30:49
        `
我吃饭冷热不忌，多烫多冷都可以吃，经常从冰箱里端出来菜和稀饭直接下肚，或者一边炒着锅里的一边吃。山东人的炸昆虫（如楼上
\@张晶
所说的）是少数我不怎么吃的东西，非要我吃我估计也能吃下，但只是觉得这玩意儿在食物充足的今天，似乎毫无必要再吃了，吃一把虫子还不如吃个鸡蛋。

感觉你们的凉菜好像是现做的。我们的这些肉鱼冻都是提前海量生产，装在直径一米多的大盆里，随正月里客人来来去去吃半个月。

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2021-06-16 06:15:43
        `
对，都是现做的。我的胃对冷的东西很敏感，一般不会吃太冷的，热的到是不忌，我属于可以守在锅边吃饺子那种。

我听家里人说，以前过年家里杀猪，会拿一部分肉炖特别大一锅萝卜，从初一开始吃到十五，到后来萝卜都酸了还在吃。不过，从我记事起好像就不这么弄了。

昆虫我也尽量不吃，以前尝试过，味道没有好到让我愿意克服心理膈应感觉的地步。以下是点重口味的，你要是对恶心的食物敏感就别看了。

我们家那边产一种蘑菇，当地叫乔巴菌。这种蘑菇非常美味，但有个特点是上边一般会长特别多的蛆，而且据说蛆多显示这种蘑菇长的好，没有蛆的反倒不敢吃。我记得小时候吃过一次这种蘑菇做的汤，汤表面浮着一层蛆，但喝起来极其鲜美。搬到县里后就没有再吃过了。

现在吃生的倒是顺口了，各海鲜，牛肉，马肉，鸡肉，鸡蛋都吃过生的了。不过对日本人民用生鸡蛋加酱油做蘸料吃肉片还是无法理解。

**Reply by yufree** [reply]
`Originally posted on 2021-06-16 13:41:45
        `
凉菜里我接受不了的是带奶酪的西式沙拉，其实除了汉堡跟披萨放奶酪外其他形式的奶酪我都吃不惯。美国会有那种把奶酪切成小块插上牙签当零食的做法，还有一片生火腿包在奶酪棒上的小吃，甚至还有油炸奶酪棒这种东西，超市里奶酪经常有一个大专柜，不同种类奶酪也有鄙视链，我是即吃不出好吃也理解不来他们对奶酪的喜爱。当然，美国人也不理解为啥有人不喜欢奶酪，他们喜欢的寿司也是要加牛油果跟奶酪做本土化改良的，有点类似粤菜里对西餐的借鉴而出现的茶餐厅菜。

**Reply by yufree** [reply]
`Originally posted by yihui on 2021-06-16 15:15:59
        `
我很耐恶心，蛆我看看没问题，因为我从小就在满是蛆的茅房拉撒，看惯了。不过要我吃的话，那恐怕也是吃不下去的。

我刚来美国的时候吃牛排只吃全熟的，后来也渐渐适应吃生牛肉了，生的更鲜嫩一些，反而觉得吃全熟的牛肉像嚼木渣渣。

**Reply by yufree** [reply]
`Originally posted by yihui on 2021-06-16 15:18:14
        `
我读博的时候在系里吃饭，就有南美同学表示惊异：你们居然不吃奶酪？感觉就像是离了奶酪没法活一样，无法想象世上竟然还有人不吃奶酪。

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2021-06-16 15:47:52
        `
哈哈哈，看来我们都差不多．我也不理解欧美一些朋友为什么那么爱吃奶酪，他们也不理解我为啥不感兴趣．牛油果寿司也吃过，不过一般是跟虾和洋葱搭配，味道不错．加奶酪的还没有吃过．

**Reply by yufree** [reply]
`Originally posted by Liechi Zhang on 2021-06-16 15:52:31
        `
小时候好像不太敏感，现在估计也吃不下了．

我现在也习惯吃半生半熟的牛排了，特别是和牛的，全烤熟了反而没有那个特别的香味和口感了．

**Comment by res-daqian** [comment]
今天读了您一天的博客，收获颇丰，看到这篇的时候，惊喜的发现我竟然同如此优秀的博主是老乡！同在海外要饭，忍不住在24年初跟博主打个招呼：~老相好！~ 老乡好！

**Reply by yufree** [reply]
老乡好！
